manticore/wasm/state.py

```diff
@@ -0,0 +1,6 @@
+from ..core.state import StateBase
+
+
+class State(StateBase):
+    def execute(self):
+        return self._platform.execute()
```

Comment:
Loud thinking: might be worth to move it to `StateBase` and just inherit from it here? It is the same in ethereum's `State` and native has a more extended impl